pkgs/tools/filesystems/fuse-archive/default.nix

```
@@ -0,0 +1,47 @@
+{ lib
+, fetchFromGitHub
+, pkg-config
+, stdenv
+, fuse
+, libarchive
+}:
```

```suggestion
{ lib
, stdenv
, fetchFromGitHub
, fuse
, libarchive
, pkg-config
}:
```

This is a bit of style. The order of parameters in the input list is: lib, stdenv,  fetchers, and all the rest in asciibetical  order.

---

Thanks, I've force-pushed this update as well.